crypto/multisig/src/lib.rs

```diff
@@ -191,13 +254,22 @@ mod test {
     /// In other places in the code we might convert to a HashSet first and then
     /// compare, but that would hide duplicate elements and we want to catch
     /// that.
+    #[track_caller]
```

Comment:
Interesting, I hadn't learned about [`#[track_caller]`](https://rustc-dev-guide.rust-lang.org/backend/implicit-caller-location.html) yet, thanks for introing me to it!